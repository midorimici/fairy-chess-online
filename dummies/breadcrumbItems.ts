export const items = [
  {
    name: 'Home',
    path: '/',
  },
  {
    name: 'Docs',
    path: '/docs',
  },
  {
    name: 'Breadcrumb',
    path: '/docs/breadcrumb',
  },
];
