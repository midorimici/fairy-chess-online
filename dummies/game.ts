export const game = {
  id: '1234567890abcdef',
  name: 'Game name',
  description:
    'Game desciption. This part may contains the description about the game or/and pieces used in the game.',
  initialBoardImage: {
    url: 'https://media.graphcms.com/zzxpvUPeT1yHQAO8vSrI',
    width: 256,
    height: 256,
  },
};
