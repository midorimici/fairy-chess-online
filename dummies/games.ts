export const games = [
  {
    id: '1234567890abcdef',
    name: 'First game',
    description:
      'Game desciption. This part may contains the description about the game or/and pieces used in the game.',
    initialBoardImage: {
      url: 'https://media.graphcms.com/zzxpvUPeT1yHQAO8vSrI',
      width: 256,
      height: 256,
    },
  },
  {
    id: '1234567890abcde0',
    name: 'Second game',
    description:
      'Game desciption. This part may contains the description about the game or/and pieces used in the game.',
    initialBoardImage: {
      url: 'https://media.graphcms.com/zzxpvUPeT1yHQAO8vSrI',
      width: 256,
      height: 256,
    },
  },
];
