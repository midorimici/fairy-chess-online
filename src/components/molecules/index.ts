export { BreadcrumbNav } from './BreadcrumbNav';
export { Container } from './Container';
export { GameList } from './GameList';
