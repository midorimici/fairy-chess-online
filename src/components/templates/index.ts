export { Game } from './Game';
export { Games } from './Games';
export { Home } from './Home';
export { NewRoom } from './NewRoom';
