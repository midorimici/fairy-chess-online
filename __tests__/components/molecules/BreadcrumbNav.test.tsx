import { render } from '@testing-library/react';
import { items } from 'dummies/breadcrumbItems';
import { BreadcrumbNav } from '~/components/molecules';

it('renders BreadcrumbNav unchanged', () => {
  const { asFragment } = render(<BreadcrumbNav items={items} />);
  expect(asFragment()).toMatchSnapshot();
});
