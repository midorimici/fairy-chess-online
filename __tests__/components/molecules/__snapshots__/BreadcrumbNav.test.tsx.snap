// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders BreadcrumbNav unchanged 1`] = `
<DocumentFragment>
  <nav
    aria-label="breadcrumb"
    class="chakra-breadcrumb css-0"
  >
    <ol
      class="chakra-breadcrumb__list css-0"
    >
      <li
        class="chakra-breadcrumb__list-item css-0"
      >
        <a
          class="chakra-breadcrumb__link css-0"
          href="/"
        >
          Home
        </a>
        <span
          class="css-0"
          role="presentation"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="ChevronRightIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
            />
          </svg>
        </span>
      </li>
      <li
        class="chakra-breadcrumb__list-item css-0"
      >
        <a
          class="chakra-breadcrumb__link css-0"
          href="/docs"
        >
          Docs
        </a>
        <span
          class="css-0"
          role="presentation"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="ChevronRightIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
            />
          </svg>
        </span>
      </li>
      <li
        class="chakra-breadcrumb__list-item css-0"
      >
        <span
          aria-current="page"
          class="chakra-breadcrumb__link css-0"
        >
          Breadcrumb
        </span>
      </li>
    </ol>
  </nav>
</DocumentFragment>
`;
